pkg/telemetrystore/telemetrystore.go

```diff
@@ -0,0 +1,32 @@
+package telemetrystore
+
+import (
+	"context"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+)
+
+type TelemetryStore interface {
+	// Returns the SigNoz Wrapper for Clickhouse
+	ClickHouse() clickhouse.Conn
```

Comment:
```suggestion
	ClickHouseDB() clickhouse.Conn
```
To keep it consistent with sqlstore.SQLStore {SQLDB} interface

Reply:
Or if you feel we can keep something more idiomatic. Any suggestions?

Reply:
Don't have an issue. Let's keep it ClicHouseDB if required we can change it later.